backport-changelog/6.6/61266.md

```diff
@@ -0,0 +1,3 @@
+https://github.com/WordPress/wordpress-develop/pull/6605
```

Comment:
file name needs to be 6605

Reply:
> Please update the content of backport-changelog/6.6/61266.md to include the core backport PR URL, or update the file name to match the core backport PR number.

Reply:
Ok I though it was the Gutenberg PR number.